examples/todo/imports/ui/login/Login.js

```diff
@@ -54,6 +54,7 @@ export class Login extends React.Component {
       {
         email: this.state.emailSubscribe,
         password: this.state.passwordSubscribe,
+        profile: { name: '' },
```

Comment:
why put it in profile? is it easy to add directly to user object?

Reply:
Just if we change the implementation of Accounts.js ... We can take a look it.